### DISCUSSION TITLE
Articles deletion : automtic purge/manual purge

### DISCUSSION THREAD

**Discussion by jediafr** [martinrotter/rssguard#860] [Q&A]
Is there a way to 'flag' or mark an article so it wont be deleted unless i do it manually ? (a bit like the star system of quiterss).

I only found importance as a marker but is it the right way ?

**Comment by martinrotter** [comment]
no articles are ever automatically deleted from RSS Guard

**Reply by jediafr** [reply]
thanks for the quick reply
